app/assets/stylesheets/desktop/login.scss

```diff
@@ -14,13 +14,16 @@
       color: rgba(var(--primary-rgb), 0.85);
     }
   }
+  h2.login-with {
+    color: var(--secondary);
+  }
   #login-buttons:not(.hidden) {
     display: flex;
     flex: 0 1 auto;
     flex-direction: column;
     align-items: stretch;
     min-height: 75px;
-    min-width: 160px;
+    min-width: 200px;
```

Comment:
Does this cause issues on small-ish window sizes?

Reply:
How small are we thinking? It currently looks fine up until about 520px, then a horizontal scroll kicks in on the modal. 

I feel this is fine for now, as we are eventually moving to full page right?